apps/src/applab/PlaySpaceHeader.jsx

```diff
@@ -59,7 +69,7 @@ var PlaySpaceHeader = React.createClass({
         <table style={{width: '100%'}}>
           <tbody>
             <tr>
-              <td style={{width: '120px'}}>{leftSide}</td>
+              <td style={{width: toggleGroupWidth}}>{leftSide}</td>
               <td style={{maxWidth: 0}}>{rightSide}</td>
```

Comment:
Weird, it does seem like maxWidth of 0 would hide this element. I assume it has something to do with table layout that it actually works. Here's how it looks when the View Data button is visible:
<img width="1645" alt="screen shot 2016-06-24 at 9 17 32 am" src="https://cloud.githubusercontent.com/assets/8001765/16343570/a86dc9da-39ec-11e6-923b-6d5b1c0805ea.png">

Since this element will be going away when we launch Firebase + Data Browser, I'm inclined not to mess with it.